sensing_launch/launch/velodyne_node_container.launch.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2020 Tier IV, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import launch
+from launch import LaunchContext
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration, PythonExpression
+from launch_ros.actions import ComposableNodeContainer, LoadComposableNodes
+from launch_ros.descriptions import ComposableNode
+import uuid
+
+
+def acceptable_unique_name(prefix):
+    id = str(uuid.uuid4())
+    # ros2 apparently doesn't accept the UUID with hyphens in node names
+    return prefix + id.replace('-', '_')
+
+
+def generate_launch_description():
+    launch_arguments = []
+
+    def add_launch_arg(name: str, default_value=None):
+        # a default_value of None is equivalent to not passing that kwarg at all
+        launch_arguments.append(DeclareLaunchArgument(name, default_value=default_value))
+
+    add_launch_arg('model')
+    add_launch_arg('launch_driver', 'True')
+    add_launch_arg('calibration')
+    add_launch_arg('device_ip', '192.168.1.201')
+    add_launch_arg('sensor_frame', 'velodyne')
+    add_launch_arg('base_frame', 'base_link')
+    add_launch_arg('container_name', 'velodyne_composable_node_container')
+    add_launch_arg('min_range')
+    add_launch_arg('max_range')
+    add_launch_arg('pcap', '')
+    add_launch_arg('port', '2368')
+    add_launch_arg('read_fast', 'False')
+    add_launch_arg('read_once', 'False')
+    add_launch_arg('repeat_delay', '0.0')
+    add_launch_arg('rpm', '600.0')
+    add_launch_arg('laserscan_ring', '-1')
+    add_launch_arg('laserscan_resolution', '0.007')
+    add_launch_arg('num_points_thresholds', '300')
+    add_launch_arg('invalid_intensity')
+
```

Comment:
There should also be following parameters: 

```suggestion
    add_launch_arg('frame_id', 'velodyne')
    add_launch_arg('gps_time', 'False')
    add_launch_arg('input_frame', LaunchConfiguration('base_frame'))
    add_launch_arg('output_frame', LaunchConfiguration('base_frame'))
```

Note that the fix for input_frame and output_frame in https://github.com/tier4/autoware_launcher.iv.universe/pull/14#discussion_r542172821 was for pointcloud_preprocessor.launch.py, but I noticed that it should also be provided for container.launch.py as well.

Reply:
@mitsudome-r I checked https://github.com/ros-drivers/velodyne/blob/ros2/velodyne_driver/config/VLP16-velodyne_driver_node-params.yaml and saw that `gps_time` is a parameter for the driver. Then I also noticed that `time_offset` and `enabled` are available but not defined here, and `pcap` is defined here but in the linked config file. Is that OK?